src/app/dashboard-widgets/applications-widget/applications-list-item/applications-list-item.component.ts

```diff
@@ -90,7 +90,6 @@ export class ApplicationsListItemComponent implements DoCheck, OnInit {
       this.inputActionDialog.proceed();
     }
   }
-
```

Comment:
why was this line removed? It is there for readability?

Reply:
Unintentional. I had changed this file. Might have left this. Let get it how it was

Reply:
Yup, I know why. It gives me linting errors without that.
93,1 trailing whitespace

Reply:
Is there a single white space in the line? If so, just remove the single space and leave the line.